tracer/test/snapshots/Iast.ExecuteCommandFromHeader.AspNetMvc5.IastEnabled.verified.txt

```diff
@@ -0,0 +1,89 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /iast/executecommandfromheader,
+    Service: sample,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: {controller}/{action}/{id},
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/ExecuteCommandFromHeader,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "COMMAND_INJECTION",
+      
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "file.txt",
+            "source": 0
+          },
+          {
+            "value": " "
+          },
```

Comment:
I assume it's correct, but what's this from? 🤔

Reply:
When building the evidence, we concat the file and the args with a whitespace separation.